controlplane/src/core/repositories/FederatedGraphRepository.ts

```diff
@@ -782,4 +788,188 @@ export class FederatedGraphRepository {
         }) as GraphApiKeyDTO,
     );
   }
+
+  public async registerClient(userId: string, federatedGraphName: string, clientName: string): Promise<string> {
```

Comment:
Please create a repository for the client db access. This is the repo for a federated graph.

Reply:
Done